source/viz.md

```diff
@@ -1031,14 +1099,15 @@ the `sort_values` function followed by the `iloc` property.  Then to help us mak
 space, we'll use horizontal bars instead of vertical ones. We do this by
 swapping the `x` and `y` variables:
```

Comment:
avoid colon before code block (please fix this everywhere in this chapter and open an issue for it for a global pass)